### PR TITLE
feat: add Celestia chain

### PR DESCRIPTION
**Pull request by jcstein**
And also update Arabica and Mocha config.

### CODE REVIEW

File: celestia/chain.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      ]
+    },
+    "codebase": {
+      "git_repo": "https://github.com/celestiaorg/celestia-app",
+      "recommended_version": "v1.1.0",
+      "compatible_versions": [
+        "v1.1.0"
+      ],
+      "genesis": {
+        "genesis_url": "https://raw.githubusercontent.com/celestiaorg/networks/master/celestia/genesis.json"
```
Comment (jcstein) [inline celestia/chain.json:30]: this is not yet live

File: celestia/chain.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          "compatible_versions": [
+            "v1.1.0"
+          ]
+        }
+      ]
+    },
+    "peers": {
+      "seeds": [
+        {
+          "id": "f97a75fb69d3a5fe893dca7c8d238ccc0bd66a8f",
```
Comment (jcstein) [inline celestia/chain.json:45]: this will need to be updated cc @sysrex for vis

File: celestia/chain.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+          "address": "celestia-blockspacerace.seed.brocha.in:30583",
```
Comment (jcstein) [inline celestia/chain.json:46]: along with address

File: celestia/chain.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          "provider": "Brochain"
```
Comment (jcstein) [inline celestia/chain.json:47]: and provider
```suggestion
          "provider": "Celestia Labs"
```

File: celestia/chain.json [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        {
+          "address": "https://api.lunaroasis.net",
+          "provider": "Celestia Labs"
+        }
+      ]
+    },
+    "explorers": [
+      {
+        "kind": "Mintscan",
+        "url": "https://mintscan.io/celestia",
```
Comment (jcstein) [inline celestia/chain.json:68]: waiting on a confirmation that this is in fact the correct link

**Comment by jcstein** [comment]
Linking [this comment](https://github.com/chainapsis/keplr-chain-registry/pull/267#issuecomment-1781392230) @JeremyParish69 - would you advise providing both svg and png? if so, how should i provide both in the config?

**Comment by JeremyParish69** [comment]
> Linking [this comment](https://github.com/chainapsis/keplr-chain-registry/pull/267#issuecomment-1781392230) @JeremyParish69 - would you advise providing both svg and png? if so, how should i provide both in the config?

If the image files have the exact same appearance, you can specify both a png and an svg formats under a single image object. e.g., https://github.com/cosmos/chain-registry/blob/ba3c760baf9f089d4a38bf422f2e4f1e06821e3e/osmosis/assetlist.json#L30-L35

File: celestia/chain.json [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "utia",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.4,
+        "average_gas_price": 0.1,
+        "high_gas_price": 0.2
+      }
```
Comment (Ryz0nd) [inline celestia/chain.json:21]: It appears there might be a typo in the `low_gas_price` value.

Comment (jcstein) [reply celestia/chain.json:21]: should be good now?

File: celestia/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "celestiatestnet",
```
Comment (adlerjohn) [inline celestia/assetlist.json:3]: ?
```suggestion
    "chain_name": "celestia",
```

File: celestia/chain.json (hunk 6 above)
Comment (liamsi) [reply celestia/chain.json:21]:
```
"low_gas_price": 0.4,
"average_gas_price": 0.1,
"high_gas_price": 0.2
```

Does not really make sense?

File: celestia/chain.json (hunk 5 above)
Comment (jcstein) [reply celestia/chain.json]: confirmed.

File: testnets/celestiatestnet2/chain.json [hunk 8]
```diff
@@ -4,38 +4,35 @@
...
   "slip44": 118,
   "fees": {
     "fee_tokens": [
       {
         "denom": "utia",
-        "fixed_min_gas_price": 0
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.4,
+        "average_gas_price": 0.1,
+        "high_gas_price": 0.2
```
Comment (Ryz0nd) [inline testnets/celestiatestnet2/chain.json:20]:
```suggestion
        "low_gas_price": 0.1,
        "average_gas_price": 0.2,
        "high_gas_price": 0.4
```

File: testnets/celestiatestnet3/chain.json [hunk 9]
```diff
@@ -8,34 +8,31 @@
...
   "slip44": 118,
   "fees": {
     "fee_tokens": [
       {
         "denom": "utia",
-        "fixed_min_gas_price": 0
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.4,
+        "average_gas_price": 0.1,
+        "high_gas_price": 0.2
```
Comment (Ryz0nd) [inline testnets/celestiatestnet3/chain.json:20]:
```suggestion
        "low_gas_price": 0.1,
        "average_gas_price": 0.2,
        "high_gas_price": 0.4
```

Comment (jcstein) [reply testnets/celestiatestnet3/chain.json:20]: thank you @Ryz0nd

File: celestia/chain.json [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+        "name": "v1.3.0",
+        "recommended_version": "v1.3.0",
+        "compatible_versions": ["v1.3.0"]
+      }
+    ]
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "f97a75fb69d3a5fe893dca7c8d238ccc0bd66a8f",
```
Comment (jcstein) [inline celestia/chain.json:42]: @smuu

File: celestia/chain.json [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+        "address": "celestia-blockspacerace.seed.brocha.in:30583",
+        "provider": "Lunar Oasis"
+      },
+      {
+        "id": "f97a75fb69d3a5fe893dca7c8d238ccc0bd66a8f",
```
Comment (jcstein) [inline celestia/chain.json:47]: @smuu i added another so you can suggest changes for the ids and addresses

File: celestia/chain.json [hunk 12]
```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "celestia",
+  "chain_id": "celestia",
+  "pretty_name": "Celestia",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "celestia",
```
Comment (JeremyParish69) [inline celestia/chain.json:8]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: celestia/chain.json [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+      {
+        "id": "f97a75fb69d3a5fe893dca7c8d238ccc0bd66a8f",
+        "address": "celestia-blockspacerace.seed.brocha.in:30583",
+        "provider": "Lunar Oasis"
+      },
+      {
+        "id": "f97a75fb69d3a5fe893dca7c8d238ccc0bd66a8f",
+        "address": "celestia-blockspacerace.seed.brocha.in:30583",
+        "provider": "Lunar Oasis"
+      }
```
Comment (smuu) [inline celestia/chain.json:50]:
```suggestion
      {
        "id": "e6116822e1a5e283d8a85d3ec38f4d232274eaf3",
        "address": "consensus-full-seed-1.celestia-bootstrap.net:26656",
        "provider": "Celestia Foundation"
      },
      {
        "id": "cf7ac8b19ff56a9d47c75551bd4864883d1e24b5",
        "address": "consensus-full-seed-1.celestia-bootstrap.net:26656",
        "provider": "Celestia Foundation"
      }
```

File: celestia/chain.json (hunk 12 above)
Comment (jcstein) [reply celestia/chain.json:8]: https://github.com/satoshilabs/slips/pull/1625 please let me know if there's anything else, thanks!

**Comment by JeremyParish69** [comment]
I can add that utia is the staking token?

**Comment by jcstein** [comment]
yes!

**Pull request merged**